.rubocop.yml

```diff
@@ -1,10 +1,271 @@
+require:
+  - rubocop-packaging
+  - rubocop-performance
+  - rubocop-rails
+
 AllCops:
   TargetRubyVersion: 2.5
+  # RuboCop has a bunch of cops enabled by default. This setting tells RuboCop
+  # to ignore them, so only the ones explicitly set in this file are enabled.
+  DisabledByDefault: true
+  Exclude:
+    - '**/tmp/**/*'
+    - '**/templates/**/*'
+    - '**/vendor/**/*'
+    - 'actionpack/lib/action_dispatch/journey/parser.rb'
+    - 'actionmailbox/test/dummy/**/*'
+    - 'actiontext/test/dummy/**/*'
+    - '**/node_modules/**/*'
+
```

Comment:
This `action*` stuff can be removed, I think.

Reply:
They are in an exclude statement, I would keep them, because we might be using them in the future and then having errors because of missing excludes would be a misery.